### PR TITLE
Add the confusion matrix to the output

### PR DESCRIPTION
**Pull request by FGRCL**

### CODE REVIEW

File: a1/create_all_output.sh [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+python a1/main.py -m basedt -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/Base-DT-DS1.txt
+python a1/main.py -m basedt -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/Base-DT-DS2.txt
+python a1/main.py -m bestdt -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/Best-DT-DS1.txt
+python a1/main.py -m bestdt -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/Best-DT-DS2.txt
+python a1/main.py -m perceptron -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/PER-DS1.txt
+python a1/main.py -m perceptron -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/PER-DS2.txt
+python a1/main.py -m basemlp -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/Base-MLP-DS1.txt
+python a1/main.py -m basemlp -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/Base-MLP-DS2.txt
+python a1/main.py -m bestmlp -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/Best-MLP-DS1.txt
+python a1/main.py -m bestmlp -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/Best-MLP-DS2.txt
```
Comment (Jacobian8) [inline a1/create_all_output.sh:12]: We really do have to have it in a csv file thought.

Comment (Jacobian8) [reply a1/create_all_output.sh:12]: I tried saving it as a csv file, and it worked but the matrix table format was very off

**Review by Jacobian8 — APPROVED** [review]
runs and looks good! 
The only thing im realizing and this has nothing to do with your PR, but we are never using the validation sets she gives us...

File: a1/create_all_output.sh (hunk 1 above)
Comment (FGRCL) [reply a1/create_all_output.sh:12]: Yeah that's why I decided not to put in a csv file. I think whoever wrote the assignment didn't think this through. We could use a separate text file to print the metrics instead. what do you think?

**Comment by FGRCL** [comment]
> runs and looks good!
> The only thing im realizing and this has nothing to do with your PR, but we are never using the validation sets she gives us...

There's a guy in the AI slack channel that talked about it, apparently he found a way to do it. My impression is that since the grid search uses k-fold, all we need to do is to merge the test set with the validation set.

**Pull request merged**